PssNationalInstance/src/main/java/com/intellisoft/pssnationalinstance/controller/DataEntryController.java

```diff
@@ -1,9 +1,6 @@
 package com.intellisoft.pssnationalinstance.controller;
 
-import com.intellisoft.pssnationalinstance.DbDataEntryData;
-import com.intellisoft.pssnationalinstance.DbVersions;
-import com.intellisoft.pssnationalinstance.FormatterClass;
-import com.intellisoft.pssnationalinstance.Results;
+import com.intellisoft.pssnationalinstance.*;
```

Comment:
Wildcard imports are generally not recommended. So try to avoid them as much as possible.

Reply:
This has been rectified. I've observed similar implementation across other controllers which has also rectified.